src/Components/WebAssembly/Sdk/integrationtests/WasmPublishIntegrationTest.cs

```diff
@@ -701,6 +705,8 @@ public async Task Publish_HostedApp_VisualStudio_WithSatelliteAssemblies()
             var existing = File.ReadAllText(blazorwasmProjFile);
             var updatedContent = @"
         <PropertyGroup>
+            <!-- Workaround for https://github.com/mono/linker/issues/1390 -->
```

Comment:
It looks like the referenced issue was fixed. Do we need this workaround because the fix isn't shipped?

Reply:
> Do we need this workaround because the fix isn't shipped?

Yup. Waiting for the fix to get to the .net sdk